conf/airframes/ardrone2_optitrack.xml

```diff
@@ -144,11 +129,51 @@
     <define name="DDGAIN_R" value="300"/>
   </section>
 
+  <section name="STABILIZATION_ATTITUDE_INDI" prefix="STABILIZATION_INDI_">
+    <!-- setpoints -->
+    <define name="SP_MAX_PHI" value="60." unit="deg"/>
```

Comment:
Well that's interesting... seeing that I got them directly from the ardrone2_indi.xml airframe file...!?